client/src/components/Ticketing/ticketingmanager/Events/showingInputContainer.tsx

```diff
@@ -151,10 +151,16 @@ const ShowingInputContainer = ({
       <div key={index} className='shadow-xl p-5 rounded-xl mb-9 bg-violet-700'>
         <label className='font-semibold text-white mb-7 mt-7  '>Show # {index + 1}</label>
         <div className='flex flex-col gap-5 mt-5 md:pr-20'>
-          <h3 className='font-semibold text-white'>Total Tickets For Showing</h3>
+          <label
```

Comment:
Thanks for switching over from `h3`s to `label`s! It's a big usability and accessibility help